### ISSUE TITLE
How to evaluate models on MSQA

### ISSUE THREAD

**Issue by Hoar012** [MSR3D/MSR3D#8]
Hi, thanks for your great work and for releasing the MSQA dataset and benchmark!

I am trying to evaluate other models on the MSQA test set. However, I noticed that some fields mentioned in the paper, such as "scene_info_json", "image_order", and "image_paths", are missing in the downloaded test data.

Could you kindly clarify:

- How to properly run evaluation on the test set using your released data?
- Whether these fields are needed for evaluation, and if so, where to find or generate them?

Any guidance or scripts you could share would be greatly appreciated. Thank you again!

**Comment by Germany321** [comment]
We upload the training and evaluation code for the baseline method, MSR3D. Do you want to evaluate other open-source MLLMs or other fine-tuned MLLMs? We can actually provide the evaluation code for GPT-4o as an example if you are interested.
In evaluation,  the object images are retrieved through "scene_id" and "object_id", which have been included in the path name of the images. While "scene_info_json", "image_order" are not required in evaluation.

**Comment by Hoar012** [comment]
Thank you for your prompt response. It would be very helpful if you could share the evaluation code for GPT-4o.
